images/ubuntu/templates/ubuntu-minimal.pkr.hcl

```diff
@@ -220,7 +256,7 @@ source "amazon-ebs" "build_image" {
   ami_virtualization_type                   = "hvm"
   ami_groups                                = var.aws_private_ami ? [] : ["all"]
   ebs_optimized                             = true
-  spot_instance_types                       = ["t3.xlarge"]
+  spot_instance_types                       = ["m7i.xlarge"]
```

Comment:
Required here or can this be a follow-up?

Reply:
Done in #115 and #116